tests/testthat/test-get_source_expressions.R

```diff
@@ -218,7 +218,10 @@ test_that("returned data structure is complete", {
 
   for (i in seq_along(lines)) {
     expr <- exprs$expressions[[i]]
-    expect_named(expr, c("filename", "line", "column", "lines", "parsed_content", "xml_parsed_content", "content"))
+    expect_named(expr, c(
+      "filename", "line", "column", "lines", "parsed_content", "xml_parsed_content", "content",
+      "xml_find_function_calls"
```

Comment:
WDYT about naming `xml_` vs. `xp_` here? I don't have an immediate preference but we do have `xp_call_name()` exported so worth considering.

Reply:
I thought about that, but we're replacing `xml_find_all()`, so I went with the `xml_` prefix.

Reply:
SGTM, thanks